doc/Type/Mu.pod6

```diff
@@ -622,27 +623,24 @@ then do this thing".  For instance,
         say "Verbose option detected in arguments";
     } # OUTPUT: «Verbose option detected in arguments␤»
 
-The C<$verbose-selected> variable in this case contains a
-L<C<Junction>|/type/Junction>, whose value is C<any(any(False, False),
-any(False, False), any(False, False), any(True, False))>. That is actually a
-I<truish> value; thus, negating it will yield C<False>. The negation of that
-result will be C<True>. C<so> is performing all those operations under the hood.
-
 =head2 method not
 
     method not()
 
-Returns a C<Bool> value representing the logical negation of an expression.
-Thus it is the opposite of C<so>.
+Evaluates the item in boolean context (and thus collapses Junctions),
+and negates the result.
```

Comment:
Repeating the same sentence again does not help. Be either exhaustive or say for instance.

Reply:
Also, what was incorrect about the original one? You are referring to a "Boolean context" that is not defined anywhere (and should be capitalized)